source/Calamari.Tests/Calamari.Tests.csproj

```diff
@@ -163,6 +163,75 @@
     <None Update="AWS\S3\Package3\file.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Fixtures\Deployment\Conventions\ConfigTransforms\config\fizz.buzz.config">
```

Comment:
Something like `Fixtures\**\ConfigTransforms\**` might work better here, as it will mean it will scoop up all of these under a single entry (and capture new ones) as opposed to having to list each fo them independently

Reply:
Argh.. something that Rider generated. Agreed it should be like that.